chapters/memory.adoc

```diff
@@ -1712,6 +1723,12 @@ ifdef::VK_EXT_metal_objects[]
     pname:exportObjectType member must: be
     ename:VK_EXPORT_METAL_OBJECT_TYPE_METAL_BUFFER_BIT_EXT
 endif::VK_EXT_metal_objects[]
+ifdef::VK_EXT_external_memory_metal[]
+  * If the parameters define an import operation and the external handle is
+    a ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLTEXTURE_BIT_EXT or
+    ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLHEAP_BIT_EXT, then
```

Comment:
For MTLHEAP, which member of VkMemoryDedicatedAllocateInfo is used? The image or the buffer? This VU should explicitly state which field of VkMemoryDedicatedAllocateInfo must be non-null.

Reply:
Removed MTLHEAP, check comment above